### ISSUE TITLE
sbt refuses to build project

### ISSUE THREAD

**Issue by ksilin** [slick/slick-codegen-example#3]
cloned project, cded into the folder, did not change anything, simply tried to run sbt

`>  sbt` results in the following error:

``` [2.2.2]
[info] Loading global plugins from ~/.sbt/0.13/plugins
[info] Loading project definition from .../slick-codegen-example/project
[info] Set current project to main (in build file:.../slick-codegen-example/)
[error] Not a valid command: warn
[error] Not a valid project ID: warn (similar: main)
[error] Expected ':' (if selecting a configuration)
[error] Not a valid key: warn (similar: run, clean, watch)
[error] warn
[error]     ^
```

`find . | grep warn` does not return anything

Using sbt 0.13.9

**Comment by matanster** [comment]
I've just built it, replacing 0.13.0 with 0.13.9 in project/build.properties before running `sbt run`. Maybe that will also solve your problem.

**Comment by SethTisue** [comment]
likely fixed by #14, if not please open a new ticket
